sdks/python/apache_beam/io/filebasedsource_test.py

```diff
@@ -20,29 +20,25 @@
 import gzip
 import logging
 import math
-import random
 import os
+import random
 import tempfile
 import unittest
 
 import hamcrest as hc
 
 import apache_beam as beam
-from apache_beam.io import filebasedsource
-from apache_beam.io import iobase
-from apache_beam.io import range_trackers
-from apache_beam.io.filesystem import CompressionTypes
-
+from apache_beam.io import filebasedsource, iobase, range_trackers
 # importing following private classes for testing
 from apache_beam.io.concat_source import ConcatSource
-from apache_beam.io.filebasedsource import _SingleFileSource as SingleFileSource
-
+from apache_beam.io.filebasedsource import \
```

Comment:
Why this change?

Reply:
default isort was 79, switched to 80 and reverted